src/lib.rs

```diff
@@ -1275,6 +1301,34 @@ fn test_small_order() {
     }
 }
 
+#[ignore]
+#[test]
+fn second_gen_nums() {
+    use blake2::{Blake2b, Digest};
+    let generator_bytes = GENERATOR.to_bytes();
+    let mut counter = 0u64;
+    let mut array = [0u8; 32];
+    loop {
+        let mut hasher = Blake2b::new();
+        hasher.update(generator_bytes);
+        hasher.update(counter.to_le_bytes());
+        let res = hasher.finalize();
+        array.copy_from_slice(&res[0..32]);
+        if AffinePoint::from_bytes(array.clone()).is_some().into()
+            && AffinePoint::from_bytes(array)
+                .unwrap()
+                .is_prime_order()
+                .unwrap_u8()
+                == 1
+        {
+            println!("POINT! {:?}", AffinePoint::from_bytes(array).unwrap());
```

Comment:
I don't see a point in leaving `println` statements. If the message is relevant to the error, it should belong to the `panic`

Reply:
Oh, this was just the quick test I did in order to find the NUMS Generator. 

Maybe I should delete it, I'm not sure at all. WDYT?

Reply:
If I keep it, I'll move the print statement inside of the panic.

Reply:
Yes, if you move the message to the panic, it is better because the error handler of the rust testing engine will display the error correctly to the user.

Reply:
Like this:
```rust
panic!("POINT! {:?} counter {}", AffinePoint::from_bytes(array).unwrap(), counter);
```